### DISCUSSION TITLE
Yolo integration

### DISCUSSION THREAD

**Discussion by Siddharth220903** [cyberbotics/wrestling#86] [Q&A] [answered]
Hi! Could you please specify is it possible to integrate yolo network with the gpu that would be used to run the controller during evaluation? Provided we have pre-trained weights for yolo.

**Comment by omichel** [comment, marked as answer]
You can integrate whatever library you need in your docker image. You should simply add the "compute" flag so that it will allow the use of the CPU for compute. Normally adding something like:
```Dockerfile
ENV NVIDIA_DRIVER_CAPABILITIES graphics,compute,utility
```
in your Dockerfile, should enable GPU compute inside your docker.
